addons/account/static/src/views/res_company_onboarding_form_controller.js

```diff
@@ -9,4 +9,10 @@ export default class CompanyOnboardingFormController extends OnboardingStepFormC
     get stepName() {
         return "account.onboarding_onboarding_step_company_data";
     }
+    get stepIsValidationConditionMet() {
+        return (async() => {
+            const [company] = await this.orm.read(this.props.resModel, [this.props.resId], ["street"]);
+            return !!(company.street && company.street.trim());
```

Comment:
If i write like this
```suggestion
            return !!company.street?.trim();
```
Then i will get error hmmm

Reply:
Of course, `false` is not `null`, my bad.